### PR TITLE
[FEAT -OKU-32]Homepage fixes

### PR DESCRIPTION
**Pull request by twentymurial33**
## Change Type
*Put an `x` in all the boxes that apply
 
- Reviews cards should be on top of the snake & ladder board[X]
- Remove the white space between Reviews and Footer [X]
- SVGs are rendering quite blurry(Water SVG doesn't look right)-[X]
- Search icon on search bar is misaligned[X]
- Make some unit tests-[]

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-32/homepage-fixes

### CODE REVIEW

File: project_app/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  presets: [['@babel/preset-env', { targets: { node: 'current' } }]],
+  env: {
+    test: {
+      plugins: ['@babel/plugin-transform-runtime']
```
Comment (idiglove) [inline project_app/babel.config.js:5]: what is this for?

File: project_app/next.config.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  swcMinify: true,
+  swcMinify: true
 };
 
 module.exports = {
+  images: {
+    unoptimized: true
```
Comment (idiglove) [inline project_app/next.config.js:9]: why do we have this?

**Review by idiglove — CHANGES_REQUESTED** [review]
some comments and questions also suggestions in discord

File: project_app/next.config.js (hunk 2 above)
Comment (twentymurial33) [reply project_app/next.config.js:9]: Removed

File: project_app/components/CardBox.js [hunk 3]
```diff
@@ -16,30 +16,47 @@ export default function CardBox() {
...
+            d="M 0 344.3 V 304.5 C 22.4563 293.784 33.2939 246.53 54.7633
+        242.453 C 80.3892 237.586 89.3651 188.589 66.6716 166.886 C 43.9777
+        145.181 37.8666 118.37 36.994 82.6228 C 36.429 59.488 15.0331 46.5173 0
+        40.2417 V 0.450623 C 19.5597 8.59641 47.5009 25.4775 48.2366 55.6283 C
+        49.3718 102.128 57.3206 137.003 86.839 165.235 C 98.5143 176.401 103.746
+        193.121 103.742 209.646 C 103.736 234.901 91.5 259.703 71.3509 263.532 C
+        43.3868 268.841 29.2896 330.468 0 344.3 Z"
+            fill="rgb(74, 164, 216)"
+          ></path>
+        </svg>
```
Comment (idiglove) [inline project_app/components/CardBox.js:45]: have this as a separate component

File: project_app/components/CardBox.js [hunk 4]
```diff
@@ -49,12 +66,32 @@ export default function CardBox() {
...
+          V 353.817 Z M 124.464 35.4471 C 123.606 35.4471 122.582 36.2451
+          121.365 37.9193 C 121.365 37.9193 111.819 55.0142 78.4007 63.0586 C
+          44.9802 71.1029 -8.48902 84.1765 30.6586 121.383 C 69.8061 158.588
+          48.8 184.733 78.4007 209.872 C 107.999 235.013 76.4881 261.156 100.361
+          301.379 C 104.439 308.252 107.877 311.337 110.773 311.337 C 124.827
+          311.337 126.14 238.704 126.14 173.672 C 126.14 102.921 132.357 35.4471
+          124.464 35.4471 Z"
+            fill="rgb(74, 164, 216)"
+          ></path>
+        </svg>
```
Comment (idiglove) [inline project_app/components/CardBox.js:94]: same here

**Review by idiglove — CHANGES_REQUESTED** [review]

File: project_app/components/CardBox.js (hunk 3 above)
Comment (twentymurial33) [reply project_app/components/CardBox.js:45]: In a separate file

File: project_app/components/CardBox.js (hunk 4 above)
Comment (twentymurial33) [reply project_app/components/CardBox.js:94]: Separated

File: project_app/components/RightContainer.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import { RightWaterContainer } from './styled';
+
+function RightContainer() {
+  return (
+    <div>
```
Comment (idiglove) [inline project_app/components/RightContainer.js:5]: do we need the parent div?

Comment (twentymurial33) [reply project_app/components/RightContainer.js:5]: No we don't removed the outer divs

**Review by idiglove — APPROVED** [review]

**Pull request merged**